docs/internals/optimizer.rst

```diff
@@ -65,20 +66,23 @@ A "runs" parameter of "1" will produce short but expensive code. The largest val
 Opcode-Based Optimizer Module
 =============================
 
-The opcode-based optimizer module operates on assembly. It splits the
+The opcode-based optimizer module operates on assembly code. It splits the
```

Comment:
This paragraph was a big block. I broke it up into what seemed to me like reasonable sub paragraphs.